static/sass/initial/_type.scss

```diff
@@ -57,30 +57,43 @@ h3 {
 a {
   #{$link} {
     @include text-underline('action');
-    color: inherit;
+    color: color('action');
     font-weight: bold;
     text-decoration: none;
     transition: color 150ms;
+
+    h1 &,
+    h2 &,
+    h3 & {
+      background-image: none; // remove link underline for headlines
+    }
   }
 
   #{$focus} {
-    color: color('action');
+    background-image: linear-gradient(color('action-hover'), color('action-hover'));
+    color: color('action-hover');
   }
 }
 
+/// @todo totally yucky, need a better class to hook into
+///   [data-module='rst-body'] is only for blog posts
+///   and no global inner-level class exists
```

Comment:
yes, but also a class to show when it is not that type of content.... or I would still need to overwrite... since I don't think I can add a :not to a parent class.


 p {
  .not-longform & {
    margin-bottom: size('shim');
    margin-top: size('shim');
  }

  .longform & {
    margin-bottom: size('gutter');
  }
}